torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -58,12 +58,6 @@ static std::vector<std::string> TORCH_NCCL_ASYNC_ERROR_HANDLING = {
     "TORCH_NCCL_ASYNC_ERROR_HANDLING",
     "NCCL_ASYNC_ERROR_HANDLING"};
 
-// Control whether dumping debug info on watchdog
-// timeout is enabled. This variable must be set together with
-// TORCH_NCCL_ENABLE_MONITORING=1 and TORCH_NCCL_TRACE_BUFFER_SIZE > 0.
-static std::vector<std::string> TORCH_NCCL_DUMP_ON_TIMEOUT = {
-    "TORCH_NCCL_DUMP_ON_TIMEOUT"};
```

Comment:
hm, change the default to 1 but don't delete the env in case someone wants to disable? we can clean up the env later.

or are you changing so that one of the other ENVs is now the way to disable? 
 - it's not OK to make the only way to disable dumping to disk be to set TRACE_BUFFER_SIZE=0, since some users might only use in-memory + http server

Reply:
Ack. I put back the flag.
We'll do a cleanup pass later.